### PR TITLE
Fixed Button Color When Code is Validated

### PR DESCRIPTION
**Pull request by Jxwoon**
See title - made changes from 5/25 Design Office Hours
Note: These colors are for Microbit

Screenshot (32)

### CODE REVIEW

**Review by jwunderl — APPROVED** [review]

**Review by jwunderl — COMMENTED** [review]
Ah I reviewed this one before I saw the pxt one; can we actually just set tutorialCodeValidationCorrectBackground and 
tutorialCodeValidationCorrectTextColor instead of having the second 'Microbit' variables? just setting:


```
@tutorialCodeValidationCorrectBackground: #cd0365;
@tutorialCodeValidationCorrectTextColor: @white;
```

and dropping in theme/style.less should work?

File: theme/style.less [hunk 1]
```diff
@@ -80,6 +80,16 @@
             background-color: darken(@mainMenuTutorialBackground, 20%);
         }
     }
+
+    .isValidated {
```
Comment (shakao) [inline theme/style.less:84]: is this necessary? it looks duplicated from pxt

**Review by shakao — CHANGES_REQUESTED** [review]
joey's comment re: removing the changes from style.less

File: theme/site/globals/site.variables [hunk 2]
```diff
@@ -84,4 +84,10 @@
...
 @serialTextColor: black;
-@serialGraphBackground: #d9d9d9;
\ No newline at end of file
+@serialGraphBackground: #d9d9d9;
+
+/*---------------------------
+   Tutorial Code Validation
+----------------------------*/
+@tutorialCodeValidationCorrectBackground: #cd0365;
+@tutorialCodeValidationCorrectTextColor: @white;
```
Comment (shakao) [inline theme/site/globals/site.variables:93]: this one is also duplicated, i think

**Review by shakao — APPROVED** [review]

**Pull request merged**